applications/netpyne/Dockerfile

```diff
@@ -16,7 +16,9 @@ RUN yarn build-dev
 ### Download on a separate stage to run in parallel with buildkit
 FROM quay.io/jupyter/base-notebook:python-3.12 AS downloads
 USER root
-RUN wget --no-check-certificate -O /nyhead.mat https://www.parralab.org/nyhead/sa_nyhead.mat
+RUN apt-get update -qq && apt-get install aria2 -y
+RUN aria2c -j10 -x 10 https://www.parralab.org/nyhead/sa_nyhead.mat
```

Comment:
download and move are better done in one single command, as otherwise the file is present in 2 separate layers, increasing the image size

Reply:
Ok, have made that change. Use of aria2c seems to increase the speed of downloading, sometimes wget would be stuck for ages with the download.